ci/requirements-py27.yml

```diff
@@ -8,3 +8,6 @@ dependencies:
   - xarray
   - matplotlib
   - dask
+  - netcdf4
+  - hdf5
+  - hdf4
```

Comment:
@pwolfram, thanks for taking the time to do a review.

My understanding is that `hdf5` *should* be needed for netcdf4.  However, it was complaining that `hdf4` was missing even when `hdf5` was present.  So I added `hdf4` as well.  If you can figure out how to get it working without `hdf4`, I'm happy to remove it.

To be honest, I don't know why the required libraries (`hdf4`, `hdf5`, or both) weren't installed automatically.  I'm just doing what it takes to fix the CI...